github_repo_loc_analyser/code_analyser.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+import subprocess
+from pprint import pprint
+
+import git
+
+# TODO: Change print to logging statements
+from github_repo_loc_analyser.data_structure import AnalysisRepo, Result
+
+
+class CodeAnalyzer:
+    WORK_DIR = "./repo/"
+    CLOCK_EXECUTABLE = "cloc"
+
+    def __init__(self, repo: AnalysisRepo):
+        self.repo = repo
+
+    def shallow_clone_repo(self):
+        print('Cloning repository ' + self.repo.get_name() + '...')
+
+        if not os.path.exists(self.WORK_DIR):
```

Comment:
Maybe make sure the WORK_DIR is deleted if it exists. Otherwise there may be an old repo in it.
Or should we do that on a higher level?